app/Party.php

```diff
@@ -677,22 +634,10 @@ public function isUpcoming()
      */
     public function isStartingSoon()
     {
-        $current_date = date('Y-m-d');
-        $event_date = $this->event_date;
-
-        if ($current_date != $event_date) {
-            return false;
-        }
-
-        if ($this->isInProgress()) {
-            return false;
-        }
-
-        $date_now = new \DateTime();
-        $event_end = new \DateTime($this->event_date.' '.$this->end);
+        $start = new Carbon($this->event_start_utc);
 
-        if ($date_now > $event_end) {
-            return false;
+        if (!$this->$this->isInProgress() && !$this->hasFinished() && $start->isCurrentDay()) {
+            return true;
         }
 
         return true;
```

Comment:
Should return false on final line here?  (Otherwise this method seems to always return true.)

Reply:
Yes.